### ISSUE TITLE
ComboBox primitive mode doesn't work with number

### ISSUE THREAD

**Issue by sigVince** [telerik/kendo-angular#461]
See plunkr :

http://plnkr.co/edit/eWrn7dFteObGMcS4c8R8?p=preview

It fails on blur, trying to toLower() a number:

```
ERROR TypeError: i.toLowerCase is not a function
    at e.handleBlur (kendo-angular-dropdowns.js:2)
```

**Comment by tapopov** [comment]
Hey @sigVince,

This should be fixed in v0.39.6.

**Comment by sigVince** [comment]
Thanks @tapopov , I am currently migrating to Angular4 and will update all my kendo-angular packages afterwards. 
I'll let you know when I'm done.
